woocommerce/wc-admin
# Automatically update versions on build

Fixes #1876

* Updates versions automatically so we don't need to update in multiple places.
* Uses the `package.json` version as the source of truth and updates the version in the plugin meta data to match this version.
* Checks the plugin version in the install script to compare against the existing installed version.

### Detailed test instructions:

1. Change the version number in `package.json`.
2. Make a temporary database change in `WC_Admin_Install` (to later note that database updates are made on version change).
3. Run `npm start` or `npm build`.
4. Note the version is updated inside `woocommerce-admin.php`.
5. Note the database updates were ran on visiting the dashboard.

### Question

I don't think the `get_plugin_data()` is a performance intensive task, however, we could also just write the version as a constant from the build script since we already have the file open to update the plugin version.  Thoughts?
